intelmq/bots/collectors/http/collector_http.py

```diff
@@ -100,5 +135,39 @@ def process(self):
                 report.add("extra.file_name", file_name)
             self.send_message(report)
 
+    def verify_signature(self, data: bytes):
+        """
+        Download signature file and verify the report data.
+        """
+
+        # get GPG parameters
+        suffix = getattr(self.parameters, "gpg_signature_suffix", ".asc")
+        keyring = getattr(self.parameters, "gpg_keyring", None)
+
+        http_url = self.parameters.http_url + suffix
```

Comment:
That won't work for URL formatting, better provide the actually used URL also as parameter to this method